examples/express/app/entry.server.tsx

```diff
@@ -123,7 +125,9 @@ function handleBrowserRequest(
           reject(error);
         },
         onError(error: unknown) {
-          console.error(error);
+          console.error(
+            (error as Error)?.stack ? (error as Error).stack : error,
+          );
```

Comment:
I think we don't need these changes anymore?

Reply:
I think it's best to keep them because it's the node/express example, so `Error.prototype.toString()` isn't patched.